internal/validations/clusterlogforwarder/validate_clusterlogforwarderspec_test.go

```diff
@@ -376,7 +376,7 @@ var _ = Describe("Validate clusterlogforwarderspec", func() {
 				},
 			}
 			verifyOutputs(cluster, client, forwarderSpec, clfStatus, extras)
-			Expect(clfStatus.Outputs["aKafka"]).To(HaveCondition("Ready", true, "", ""))
+			Expect(clfStatus.Outputs["aKafka"]).To(HaveCondition("Ready", false, "Invalid", "URL is required for output type kafka"))
```

Comment:
This check seems wrong - a URL is _not_ required for kafka, the brokers can be specified in the URL _or_ in the kafka.brokers field.